### ISSUE TITLE
Table naming pattern

### ISSUE THREAD

**Issue by tischi** [hci-unihd/batchlib#127]
@constantinpape 
Sorry, small thing, but for next run, could you maybe change this:
```
20200417_132123_311_cell_table_serum_IgG.xlsx*
20200417_132123_311_image_table.xlsx*
20200417_132123_311_well_table.xlsx*
```

to 
```
20200417_132123_311_cell_serum_IgG_table.xlsx
```

would be more consistent naming scheme and a bit easier to parse.
that is:

```
PLATENAME_CONTENT_table.xlsx
```

**Comment by constantinpape** [comment]
Yes, makes a lot of sense.

**Comment by constantinpape** [comment]
Implemented in #130. Gonna close once it's merged.
